beacon_node/beacon_chain/src/test_utils.rs

```diff
@@ -54,8 +54,8 @@ where
 
 impl<L, E> BeaconChainTypes for CommonTypes<L, E>
 where
-    L: LmdGhost<MemoryStore, E>,
-    E: EthSpec,
+    L: LmdGhost<MemoryStore, E> + 'static,
+    E: EthSpec + 'static,
```

Comment:
`EthSpec` implies `'static` and `Clone` so we don't need it again here (sorry, I'm on a vendetta against trait bounds)

Reply:
Thanks!! I'll go squash 'em!